test/cmdlineTests/model_checker_engine_none/input.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+pragma experimental SMTChecker;
+contract test {
+	// `y` is here just to generate a warning, otherwise the command line test
+	// expectation fails with "Obtained no output requested".
+    function f(uint x, uint y) public pure {
```

Comment:
You could also just remove the version pragma and license comment and get a warning that's cearly unrelated to the example.

Reply:
Right, did that instead.